filters/filters.go

```diff
@@ -107,12 +109,18 @@ type FilterContext interface {
 	// Allow filters to collect metrics other than the default metrics (Filter Request, Filter Response methods)
 	Metrics() Metrics
 
-	// Allow filters to add Tags, Baggage to the trace or set the ComponentName.
+	// Allow filters to add Tags, Baggage to open tracing trace or set the ComponentName.
 	Tracer() opentracing.Tracer
```

Comment:
Is it okay to still keep FilterContext.{Tracer,ParentSpan} methods returning "deprecated" opentracing types exposed?

Reply:
Yeah, the idea is to keep supporting both for the next year because currently all custom filter implementations that use tracing are still using open telemetry, we don't want to break backward compatibility and leave some time for users change their code base. I think something that we could improve is add a warning to these functions when used, so skipper users can see warnings in the logs that this functionality will be deprecated soon.